Aquality.Selenium.Template/Aquality.Selenium.Template/Constants/ProjectConstants.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+
+namespace Aquality.Selenium.Template.Constants
+{
+    public class ProjectConstants
+    {
+        public static readonly TimeSpan ShortTimeout = TimeSpan.FromSeconds(5);
+        public const int PollingInterval = 1;
```

Comment:
such hardcoded constants are antipattern, names like "ShortTimeout" and "Timeout" and "ConstantText" are antipatterns as well, I'm sure we don't want them in the template project

Reply:
Updated